cupyx/_ufunc_config.py

```diff
@@ -1,16 +1,51 @@
 import contextlib
 import threading
 
-config = threading.local()
-config.divide = None
-config.over = None
-config.under = None
-config.invalid = None
-# In favor of performance, the `devInfo` input/output from cuSOLVER routine
-# calls that is necessary to check the validity of the other outputs, are
-# ignored, as D2H copy incurring device synchronizations would otherwise be
-# required.
-config.linalg = 'ignore'
+_config = threading.local()
+
+
+def get_config_divide():
+    try:
+        return _config.divide
+    except AttributeError:
+        _config.divide = None
```

Comment:
Nitpick but how about this? It's concise and avoids repetition (in terms of both code and computation).
```suggestion
        return _config.divide = None
```

Reply:
Syntax error 😢but I liked it.

Reply:
I see. How about this, then? I don't like repetition.
```py
try:
    value = _config.divide
except AttriuteError:
    value = _config.divide = None
return value
```